app/models/openscap_result.rb

```diff
@@ -0,0 +1,68 @@
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/benchmark'
+
+class OpenscapResult < ApplicationRecord
+  belongs_to :container_image
+  has_one    :binary_blob,           :dependent => :destroy, :autosave => true, :as => :resource
+  has_many   :openscap_rule_results, :dependent => :destroy, :autosave => true
+
+  OpenSCAP.oscap_init
```

Comment:
`OpenSCAP.oscap_init` has very small foot print. However, I would also vote for running it only just before specific openscap operation (like creating ARF). The other operations, actually, may not need this init to be called. If you can, please run `OpenSCAP.oscap_cleanup()` afterwards.